.rubocop_todo.yml

```diff
@@ -77,9 +143,9 @@ Style/RegexpLiteral:
   Exclude:
     - 'test/test_helper.rb'
 
-# Offense count: 542
+# Offense count: 5994
```

Comment:
Something seems wrong here.

Reply:
Huh.  Not sure why that's happening as it is mostly just moving files... I'll investigate.